### PR TITLE
[REF] web: remove ControlPanelBottom

### PR DESCRIPTION
**Pull request by aab-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/search/control_panel/control_panel.xml [hunk 1]
```diff
@@ -16,42 +16,33 @@
                 </div>
             </div>
             <div t-if="display['bottom']" class="o_cp_bottom">
-                <t t-if="display['bottom-content']">
```
Comment (Polymorphe57) [inline addons/web/static/src/search/control_panel/control_panel.xml:19]: there are simpler alternatives?

Comment (aab-odoo) [reply addons/web/static/src/search/control_panel/control_panel.xml:19]: Simple, not necessarily simpler (from the dashboard pov)

**Comment by ged-odoo** [comment]
robodoo r+

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**